README.md

```diff
@@ -52,8 +54,8 @@ var foods = [ ["apple","banana","strawberry"], ["pizza","fries","hamburger"] ];
 
 How would I go about accessing the string "pizza" in the above array?
 
-```js
-// write code here
+```
+var meal = foods[3];
```

Comment:
in order to access the value `"pizza"` in the given variable `foods`, we have to access the second inner array, then `pizza` is the first item in that array. i.e `foods[1][0]`